research/object_detection/dataset_tools/create_ava_tf_record_for_context.py

```diff
@@ -0,0 +1,403 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
```

Comment:
When you get a chance, please respond to the comments and resolve them, that way its easier to understand the fixes you made.